roles/system_settings/README.md

```diff
@@ -10,103 +10,114 @@ N/A
 
-- **system_settings_dns_domains**:
-  - List of DNS domains to add/set.
+- **system_settings_dns_domains** (list(str))
+    - List of DNS domains to add/set.
 
-- **system_settings_dns_mode_append**
-  - If `true` items from `system_settings_dns_domains` and `system_settings_dns_servers` will be added to already configured DNS domains/servers. If `false` domains/servers will be overridden.
+- **system_settings_dns_mode_append** (bool)
+    - If `true` items from `system_settings_dns_domains` and `system_settings_dns_servers` will be added to already configured DNS domains/servers. If `false` domains/servers will be overridden.
 
-- **system_settings_dns_hostname**:
-  - Set the hostname of the vCenter.
+- **system_settings_dns_hostname** (str)
+    - Set the hostname of the vCenter.
 
 #### NTP
 Note: NTP service will be restarted if configuration is changed.
 
-- **system_settings_timesync_mode**:
-  - Set time synchronization mode.
+- **system_settings_timesync_mode** (str)
+    - Set time synchronization mode.
 
-- **system_settings_ntp_servers**:
-  - List of NTP servers. This method updates old NTP servers from configuration and sets the input NTP servers in the configuration. If NTP based time synchronization is used internally, the NTP daemon will be restarted to reload given NTP configuration. In case NTP based time synchronization is not used, this method only replaces servers in the NTP configuration.
+- **system_settings_ntp_servers** (list(str))
+    - List of NTP servers. This method updates old NTP servers from configuration and sets the input NTP servers in the configuration. If NTP based time synchronization is used internally, the NTP daemon will be restarted to reload given NTP configuration. In case NTP based time synchronization is not used, this method only replaces servers in the NTP configuration.
 
-#### Proxy
+#### Appliance Proxy
 
-- **system_settings_noproxy**:
-  - List of hosts that should be ignored by proxy configuration.
+- **system_settings_noproxy** (list(str))
+    - List of hosts that should be ignored by proxy configuration.
 
-- **system_settings_proxy**:
-  - A list of proxy configurations.
-
-    Proxy options:
-    * enabled: Required. Define if this proxy configuration should be enabled.
-    * url: Required. Define the URL of the proxy server (including protocol ie. http://...).
-    * port: Required. Define the port of the proxy server.
-    * protocol: Required. Define the protocol of the proxy server(FTP, HTTP, HTTPS).
-    * state: Define if the proxy configuration should be `present` or `absent`.
-    * username: Define `username` for the proxy server if proxy requires authentication.
-    * password: Define `password` for the proxy server if proxy requires authentication.
+- **system_settings_proxy** (list(dict))
+    - A list of proxy configurations.
+    - **Elements**
+        - enabled: bool, required, Define if this proxy configuration should be enabled.
```

Comment:
```suggestion
        - enabled - bool, required, Define if this proxy configuration should be enabled.
```

Reply:
i noticed that other 'elements' sections use the format
\- `varname`: 

so i updated this section to match that format